CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to _ReadTools_
+
+The following is a set of guidelines to contribute to SwimPool.
+There is no rule that cannot be broken, so feel free to propose
+any change to this document.
+
+#### Table Of Contents
+* [Code of conduct](#code-of-conduct)
+* [How to contribute](#how-to-contribute)
+  - [Bug reports](#bug-reports)
+  - [Suggesting enhancements/features](#suggesting-enhancements-and-features)
+  - [Code contributions](#code-contributions)
+* [Recognition model](#recognition-model)
+  - [Become a maintainer](#become-a-maintainer)
```

Comment:
Recognition model does not exists, neither become a maintainer. Remove from TOC

Reply:
Maybe we should add a recognizition model by adding a `CONTRIBUTORS` file, although the copyright holder is going to be still the same.

Reply:
Added a recognition model.